src/pages/Dashboard/DashboardSidebar.tsx

```diff
@@ -64,7 +63,6 @@ export const DashboardSidebar = memo(() => {
   return (
     <Flex width='full' flexDir='column' gap={6}>
       <PromoCard data={promoData} />
-      <EligibleCarousel />
```

Comment:
q: have we confirmed this goes away as well?

Reply:
Yes in private